content/en/_index.md

```diff
@@ -204,11 +237,18 @@ See https://cloud.google.com/compute/docs/quickstart-client-libraries before run
  * before running the code snippet.
  */
 {{< /tab >}}
+{{< tab header="Terraform" >}}
+// [START product_example]
+#   Triggers DAGs Using Cloud Function and Pub/Sub Messages with Terraform
```

Comment:
Other language tabs include an example linking to prerequisite documentation such as

```
 * See https://cloud.google.com/compute/docs/quickstart-client-libraries
 * before running the code snippet.
```

Reply:
Fixed